spec/index.bs

```diff
@@ -6558,23 +6558,25 @@ pipeline, and have the following members:
                 [=Device timeline=] |initialization steps|:
 
                 1. Let |limits| be |this|.{{GPUObjectBase/[[device]]}}.{{device/[[limits]]}}.
+                1. Let |bindGroupLayouts| be a copy of |descriptor|.{{GPUPipelineLayoutDescriptor/bindGroupLayouts}}
+                1. For each |bindGroupLayout| in |bindGroupLayouts|:
+                    1. If |bindGroupLayout| is `undefined` or [=list/empty=], set |bindGroupLayout| to `null`
```

Comment:
This looks like it does nothing because it overwrites a local variable.
```suggestion
                1. For each |i| in the [=indices=] of |bindGroupLayouts|:
                    1. If |bindGroupLayouts|[|i|] is `undefined` or [=list/empty=], set |bindGroupLayouts|[|i|] to `null`.
```